src/codegen/doc/mod.rs

```diff
@@ -254,8 +254,10 @@ fn create_object_doc(w: &mut dyn Write, env: &Env, info: &analysis::object::Info
     };
 
     for function in functions {
+        let configured_functions = obj.functions.matched(&function.name);
```

Comment:
IMHO function name here replaced already, so no one will be found here

Reply:
You fell in the same trap as I did. :)

Here is a `library::Function` list, whereas I make the change inside the `analysis::functions::Info` type. So at this point, `function.name` is still correct, which is why I need to get the `rename` thing [here](https://github.com/gtk-rs/gir/pull/873/files#diff-30816be45ddaf9346e6d262a214fb1b4R273).

Reply:
You right :wink:
Then only need some formating